repl/src/main/scala/ammonite/repl/interp/Preprocessor.scala

```diff
@@ -143,7 +145,13 @@ object Preprocessor{
           "Don't know how to handle " + code
         )
       }
+    }
 
+    def script(stmts: Seq[Seq[String]], wrapperId: String): Res[Seq[Preprocessor.Output]]={
+      val results = stmts map (apply(_,wrapperId))
```

Comment:
Use method syntax (e.g. `.map`) rather than infix syntax except for operators or chains of symmetric methods (e.g. `orElse`)